doc/api-extensions.md

```diff
@@ -2363,7 +2363,6 @@ This API extension indicates that the `/1.0/instances/{name}/uefi-vars` endpoint
 This API extension allows newly created VMs to have their `migration.stateful` configuration key automatically set
 through the new server-level configuration key `instances.migration.stateful`. If `migration.stateful` is already set at the profile or instance level then `instances.migration.stateful` is not applied.
 
-## `access_management`
+## `vm_disk_io_limits`
```

Comment:
Is it intentional that you removed access_management?

Reply:
Definitely not, thanks for spotting. Fixed now.